gradescope_utils/autograder_utils/GradescopeRunner.py

```diff
@@ -0,0 +1,18 @@
+import sys
+from django.test.runner import DiscoverRunner
+from gradescope_utils.autograder_utils.json_test_runner import JSONTestRunner
+
+
+class GradescopeRunner(DiscoverRunner):
```

Comment:
This looks good! Could you disambiguate this slightly and call it 
```suggestion
class GradescopeDjangoRunner(DiscoverRunner):
```
Thanks!